package.json

```diff
@@ -158,7 +162,7 @@
   "bundlesize": [
     {
       "path": "./dist/styled-components.min.js",
-      "threshold": "14.75kB"
+      "threshold": "14.9kB"
```

Comment:
😢

Reply:
Some tweaks brought the bump down to `14.84kB`, but w/o unrelated refactors I'm not sure if I can do any better at the moment